e2e/testdata/gen-certs.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env sh
+
+GO111MODULE=off go get -u github.com/dmcgowan/quicktls
+cd notary || exit 1
```

Comment:
`set -eu` instead of `exit 1`?

Reply:
Ah, yes, I think that works as well; this was mostly to satisfy the linter 😅